### DISCUSSION TITLE
关于某度贴吧里的PCL2吧

### DISCUSSION THREAD

**Discussion by SqrtTan** [Meloong-Git/PCL#2640] [讨论]
我今天在某度贴吧的PCL2吧里面回复了几十条帖子
其中一半都是不更新的，遇到问题就在吧里喊求助，死活不看github的
还有一位PCL2报错，结果发帖时发了一张报错界面的截图的（话说不是写了发错误报告的吗）

我基本就发了一个Issues的链接，各位有功夫要不也去帮帮那边的几位仁兄？

**Comment by iPodtouch6557** [comment]
来自 #2572 

> 世界80亿人，林子大了啥鸟都有（
> 很难保证每个人都有那么高的素质

另外，这个单词拼写为 i s s u e，不是Iusse

**Reply by SqrtTan** [reply]
怪不得我输入法总是不给我拼这个单词……

**Comment by so1ve** [comment]
他都去贴吧了你还能指望他做什么。。。有时间上贴吧怎么不花时间看看如何报告问题，真搞不懂
